.github/workflows/teardown-frontend-dev.yml

```
@@ -0,0 +1,332 @@
+            echo "❌ Confirmation failed. Expected: 'I UNDERSTAND', Got: '$CONFIRM_INPUT'"
+            echo "This is a safety measure to prevent accidental environment destruction."
+            exit 1
+          fi
+
+          echo "✅ Confirmation validated"
+
+      - name: Check code ownership
+        id: codeowner-check
+        uses: ./.github/actions/check-codeowner
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.7)

input "github-token" is not defined in action "Check Code Owner" defined at "./.github/actions/check-codeowner". available inputs are "base-ref", "mode"

(action)